agent/tool-scripts/base-tool

```diff
@@ -191,12 +191,20 @@ esac
 
 function usage {
 	printf -- "The following options are available:\n\n"
+	printf -- "Operational options - do *not* specify any of these when you register a tool:\n\n"
 	printf -- "\t--install                 verify this perf tool will work\n\n"
 	printf -- "\t--start|stop|postprocess  start/stop/postprocess the data collection\n"
 	printf -- "\t-d str|--dir=str          directory to store data collection (required)\n"
+
+	printf -- "\nTool-specific options - these should be specified when you register a tool:\n\n"
+
 	case "${tool}" in
-	blktrace|bpftrace|kvmtrace|lockstat|oc|perf|strace|systemtap|tcpdump)
-		:  # These tools don't use interval
+	lockstat|strace)
+		:  # These tools don't take *any* tool-specific options
+		printf "No tool-specific options\n"
+		;;
+	blktrace|bpftrace|kvmtrace|oc|perf|systemtap|tcpdump)
+		:  # These tools don't use interval but take other tool-specific options
 		;;
 	*)
 		printf -- "\t--interval=int            number of seconds between each data collection (optional, default is %s seconds)\n" "${def_interval}"
```

Comment:
The text printed at line 204 should, arguably, start with a tab character, like the other lines (like line 210) do.